docs/user/trusted-publishers/adding-a-publisher.md

```diff
@@ -97,7 +97,12 @@ each.
 
 === "ActiveState"
 
-    TODO
+    For ActiveState, you need to provide the project name on ActiveState's Platform,
```

Comment:
```suggestion
    For ActiveState, you need to provide the name of the ActiveState project,
```
I think this makes it a bit more clear which project name this is.

Reply:
I agreed!